### PR TITLE
Release v0.3.50

### PR DESCRIPTION
**Pull request by dmanjunath**

### CODE REVIEW

**Comment by dmanjunath** [comment]
^ This alert by gitguardian is not an issue, it's flagging the wormhole contract address

**Review by cheran-senthil — APPROVED** [review]

File: audius/creator-node/creator-node-deploy-backend.yaml [hunk 1]
```diff
@@ -30,7 +30,7 @@ spec:
       - name: backend
         imagePullPolicy: Always
         
-        image: audius/creator-node:89aa525936bd1698cbd34dc354b2f557b897e0f4
+        image: audius/creator-node:96517412a87868f4ac3e2ea5ce255fb7e828731b
```
Comment (csjiang) [inline audius/creator-node/creator-node-deploy-backend.yaml:33]: qq where are these hashes from? they don't seem to line up w[ any recent git SHAs](https://github.com/AudiusProject/audius-protocol/commits/master) in the repo?

Comment (cheran-senthil) [reply audius/creator-node/creator-node-deploy-backend.yaml:33]: https://github.com/AudiusProject/audius-protocol/commit/96517412a87868f4ac3e2ea5ce255fb7e828731b

Comment (csjiang) [reply audius/creator-node/creator-node-deploy-backend.yaml:33]: my brain grows 🧠

**Review by csjiang — APPROVED** [review]

**Pull request merged**